src/Shape/Box.js

```diff
@@ -47,6 +47,7 @@ export default class Box extends BaseClass {
     };
     this._rectWidth = constant(50);
     this._whiskerConfig = {};
+    this._whiskerEndpoint = [];
```

Comment:
You need to initialize this variable in render (line 205 would work). Otherwise, each subsequent render will be adding duplicates to this array.